src/Accordion.php

```diff
@@ -170,9 +184,10 @@ public function renderItem($header, $item, $index)
             $options = ArrayHelper::getValue($item, 'contentOptions', []);
             $options['id'] = $id;
             Html::addCssClass($options, ['widget' => 'collapse']);
-            if ($index === 0) {
```

Comment:
Please watch line 190/205 as well (`aria-expanded`)

Reply:
is it like this? Not really sure about this.
```php
'aria-expanded' => ($index === 0) ? 'true' : 'false',
```

Reply:
```php
'aria-expanded' => ($this->_isAccordionExpanded($item)) ? 'true' : 'false',
```